go/keybase/main.go

```diff
@@ -47,6 +47,9 @@ func keybaseExit(exitCode int) {
 }
 
 func main() {
+	logger.EnableBufferedLogging()
```

Comment:
Don't need this

Reply:
why not? we only call it in kbfs dont' we want buffered logging in the service?

Reply:
It gets turned on in `SetLogFileConfig`